corehq/apps/users/views/mobile/users.py

```diff
@@ -769,8 +770,8 @@ def create_mobile_worker(self, in_data):
             }
 
         self.request.POST = form_data
-        is_valid = lambda: self.new_mobile_worker_form.is_valid() and self.custom_data.is_valid()
-        if not is_valid():
+
+        if not (self.new_mobile_worker_form.is_valid() or self.custom_data.is_valid()):
```

Comment:
```suggestion
        if not (self.new_mobile_worker_form.is_valid() and self.custom_data.is_valid()):
```
They both have to be valid for this to request to be considered successful

Reply:
Ah you're right. It must be either `if not self.new_mobile_worker_form.is_valid() or not self.custom_data.is_valid()`, or ` if not (self.new_mobile_worker_form.is_valid() and self.custom_data.is_valid())`. Ima keep it to the `and` version, since it was that. Thanks for catching this